deploy/ansible/vagrant.yml

```diff
@@ -0,0 +1,30 @@
+# Create an account for the currently logged-in user, and copy their public key
+# to the server. This makes it possible to ssh directly, instead of using the
+# "vagrant ssh" command, and run other playbooks without specifying a user or
+# private key on the command line. Additionally, "sudo" will be usable without
+# a password, for convenience.
+
+- hosts: vagrant
+  become: yes
+  become_method: sudo
+  tasks:
+    - name: ensure local user is synced
+      user:
+        state: present
+        name: "{{ lookup('env', 'USER') }}"
+        shell: /bin/bash
+        groups: sudo
+      register: user
```

Comment:
@tkellen I don't see any warnings for registering a variable in this way. It seems like the "bare variables" warning has to do with referencing an already-defined variable. Are you sure this needs to be changed?